docs/libs/rlottie.rst

```diff
@@ -2,7 +2,7 @@
 Lottie player
 =============
 
-Allows to use Lottie animations in LVGL. Taken from this `base repository <https://github.com/ValentiWorkLearning/lv_rlottie>`__
+Allows to use Lottie animations in LVGL. Taken from `lv_rlottie <https://github.com/ValentiWorkLearning/lv_rlottie>`__.
```

Comment:
Grammar is wrong. I have to look at the entire source file to get the context it which it is being used. I am thinking it should be along the lines of 

> Allows playing Lottie animations in LVGL.

Reply:
fixed